CONTRIBUTING.md

```diff
@@ -83,7 +83,10 @@ There could be times when there is a need for a more rich development environmen
 
 ## Coding Guidelines
 
-All code is formatted with `prettier` using the configuration in the repo. If possible we recommend configuring your editor to format automatically, but you can also use the `yarn prettier --write <file>` command to format files.
+To keep the codebase consistent and maintainable, we have a some cross workspace tolling in place:
+
+- `yarn`: is the package manager used for all workspaces. We will regularly update the yarn version to keep up with the latest features and bug fixes. This version is managed in the root `package.json` and `.yarnrc.yml` files and should not be locked to a different version in any workspace. Updating the yarn version could imply changes on all the `yarn.lock` files.
```

Comment:
@awanlin does this looks like a good place to mention this, in the issue you mentioned to document the policy and i between this and create a different file for Coding Guidelines. WDYT?

Reply:
This works for me, I had thought about under - https://github.com/backstage/community-plugins/tree/main/docs - but this seems like a good place. 👍

Reply:
yes i thought on that too, maybe even together with the Version bump documentation, if we decide to write the how to bump yarn version everywhere 🤔